src/routes/jira/atlassian-connect/jira-atlassian-connect-get.ts

```diff
@@ -236,7 +236,8 @@ const	modules = {
 };
 
 export const JiraAtlassianConnectGet = async (_: Request, res: Response): Promise<void> => {
-	modules.jiraDevelopmentTool.actions = await defineJiraDevelopmentToolModuleActions();
+	const { jiraHost } =  res.locals;
+	modules.jiraDevelopmentTool.actions = await defineJiraDevelopmentToolModuleActions(jiraHost);
```

Comment:
Just to confirm, if we change the ff, user is required to uninstall/re-install the app the make the change effective as this description will only be read once during the app installation?

Reply:
No, the connect team said these changes are automatically detected.